### DISCUSSION TITLE
Query Fields within Field Groups

### DISCUSSION THREAD

**Discussion by HarunKilic** [directus/directus#7782] [Archive]
Hi guys, 

How can i query Fields that is within Field Groups by API? 
eg. i have a **field group** called _details_. 
I want to get all the **fields** that is within this **group**.

Thanks!

**Comment by benhaynes** [comment]
Field Groups are not structural, so they don't change the API response (only the App GUI). That said, there may ways we can partially support this in the future... such as new syntax in the `fields` parameter. CC @rijkvanzanten

**Reply by HarunKilic** [reply]
It would be nice to group fields in a structured way on custom application also.

**Reply by benhaynes** [reply]
As I said, this is already possible within the app. Groups are a new data type, with current support for raw, accordion, and detail.

**Reply by HarunKilic** [reply]
Yes, but i was thinking about same possibility with API.
